### PR TITLE
Add mirror sensors to KBOMirrorHEStates

### PR DESCRIPTION
**Pull request by nrwslac**
## Description

- add mirror sensor component to `KBOMirrorHE`
- this will affect:
```
| name             | mr3k2_kbh
| name             | mr4k2_kbv
| name             | mr2k4_kbo
| name             | mr3k4_kbo

```
All tested locally.
## Motivation and Context

- RTDs installed on mirror

## How Has This Been Tested?

- ran locally and launched screens
## Where Has This Been Documented?

https://jira.slac.stanford.edu/browse/ECS-6616
https://jira.slac.stanford.edu/browse/ECS-6705

cooling-readouts
## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive docstrings, including context and API
- [ ] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: docs/source/upcoming_release_notes/1304-chin-rtds.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+1304 chin-rtds
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1304-chin-rtds.rst:1]: like the chin on my face?

**Review by tangkong — COMMENTED** [review]
I have this mental image of mirrors being little faces with chin-guards now.  

A few nitpicks here, and I do want to call out that the component names have changed, breaking the previous interface.  I expect this to be fine, but have no real way of knowing or testing this (read: I don't know what's out in the field that may rely on these devices)

File: docs/source/upcoming_release_notes/1304-chin-rtds.rst (hunk 1 above)
Comment (nrwslac) [reply docs/source/upcoming_release_notes/1304-chin-rtds.rst:1]: wait till you meet the mirrors with a tail.

File: docs/source/upcoming_release_notes/1304-chin-rtds.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+#################
+
+API Breaks
+----------
+- N/A
```
[the remark this answers is not in the document]
Comment (nrwslac) [reply docs/source/upcoming_release_notes/1304-chin-rtds.rst:6]: This is true. In this case, we should be clear. The `FFMirrorZ` mirrors are limited to MR4/5K4KBO in TMO; they haven't been fully commissioned yet afaik. I'll update the docs.

[the remark this answers is not in the document]
Comment (nrwslac) [reply docs/source/upcoming_release_notes/1304-chin-rtds.rst]: update this as an API break in detail.

File: docs/source/upcoming_release_notes/1304-chin-rtds.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+---------------
+- `KBOMirrorHE` gets 2 new RTD readouts for RTDs installed on the mirror inside vaccum.
+
+New Devices
+-----------
+- N/A
+
+Bugfixes
+--------
+- reorder some components in `KBOMirrorHE`
```
[the remark this answers is not in the document]
Comment (nrwslac) [reply docs/source/upcoming_release_notes/1304-chin-rtds.rst]: Moved to maintenance and added more details.

**Comment by nrwslac** [comment]
> I have this mental image of mirrors being little faces with chin-guards now.
> 
> A few nitpicks here, and I do want to call out that the component names have changed, breaking the previous interface. I expect this to be fine, but have no real way of knowing or testing this (read: I don't know what's out in the field that may rely on these devices)

I think I addressed all concerns here, thank you for reviewing!

**Comment by ZLLentz** [comment]
I greatly appreciate your inclusion of the screenshots into the pr description

**Review by ZLLentz — APPROVED** [review]

File: pcdsdevices/mirror.py [hunk 4]
```diff
@@ -1154,6 +1154,10 @@ class KBOMirrorHE(KBOMirror):
     cool_flow1 = Cpt(EpicsSignalRO, ':FWM:1_RBV', kind='normal')
     cool_press = Cpt(EpicsSignalRO, ':PRSM:1_RBV', kind='normal')
 
+    #
```
Comment (ZLLentz) [inline pcdsdevices/mirror.py:1157]: nitpick
```suggestion
```

Comment (nrwslac) [reply pcdsdevices/mirror.py]: Where'd this little dude come from. Updated

**Review by tangkong — APPROVED** [review]
👍

**Review by ZLLentz — APPROVED** [review]

**Pull request merged**